Fixed & updated Unity docs for road to 0.1.0 (#3603)
getsentry/sentry-docs: src/platforms/unity/configuration/event-debouncing.mdx

---

> title: Event Decouncing

Reviewer: ```suggestion
title: Event Debouncing
```

description: "Learn how the SDK handles high frequencies of events."
sidebar_order: 20
---

> Because gameplay updates at a very high frequency of 60 times a second or even higher a debouncing mechanism is in place. Every type of log has its own time slot. If you trigger too many events of the same type in quick succession Sentry will capture the first of its kind and ignore (debounce) the others.

Reviewer: ```suggestion
Because gameplay updates at a very high frequency of 60 times a second or even higher, a debouncing mechanism is in place. Every type of log has its own time slot. If a game triggers too many events of the same type in quick succession, Sentry will only capture the first of its kind, then ignore (debounce) the other events of the same type.
```
